docs/userguides/searches.md

```diff
@@ -79,6 +79,22 @@ for event in file_events:
     print(event["md5Checksum"])
 ```
 
+If the number of events exceeds 10,000 against a query, use `securitydata.SecurityModule.search_all_file_events()`:
+
+```python
+
+query = FileEventQuery(ExposureType.eq(ExposureType.REMOVABLE_MEDIA))
+token = ""
```

Comment:
Because of how we made it, this still works when token is initialized to `None`. It might be less confusing if we do that or else they may think that it is required to use empty string at the start.

Reply:
Changed.